### DISCUSSION TITLE
Ability to sort kanban columns by fields in projects beta

### DISCUSSION THREAD

**Discussion by matschaffer** [community/community#7129] [Projects and Issues] [answered]
This is for https://github.com/features/issues/

Table views have a sort option:
![Screen Shot 2021-11-02 at 12 32 35](https://user-images.githubusercontent.com/690/139782705-fe2edbd0-acc2-4df2-ae9d-ab6d4b1ad10f.png)

It'd be great if kaban had the same.

Maybe something in the ... menu here. Would need to decide how to work in field selection of course. I think it'd be okay if the sortable fields were defined as a few setting. Mostly we'll want to sort by "priority" of our use case.

![Infrastructure_Monitoring_UI](https://user-images.githubusercontent.com/690/139783044-bfb6f495-3e44-404a-8b61-453e2460f592.png)

**Comment by jayaramkasi** [comment]
Agreed. Sorting and Grouping are both needed in Kanban view, so that we can create swimlanes.

**Comment by Mel0n7** [comment]
This would be so nice for my project! For my use case I want to sort by date added to column.

**Comment by rileybroughten** [comment, marked as answer]
This is supported! Check out the changelog announcement [here](https://github.blog/changelog/2023-01-26-github-projects-january-26th-update/#sorting-by-field-values-on-the-board-layout) 👍
